wherehows-web/app/constants/dataset-aclaccess.ts

```diff
@@ -0,0 +1,132 @@
+import { capitalize } from '@ember/string';
+import { IpageConcent, IpageState, IAclInfo, Iprincipal } from 'wherehows-web/typings/api/datasets/aclaccess';
+
+/**
+ * Defined the method to gnerate the static resource of ACL page based on the current user
+ * @param {string} userName
+ * @return {IpageConcent} pageContent
+ */
+const pageContent = (userName = ''): IpageConcent => {
+  userName = capitalize(userName);
+  return {
+    success: {
+      info: `${userName}, you have access to this data`,
+      requestInfo: 'Congrats! Your request has been approved!',
+      requestMessage: 'You now have a access to this data',
```

Comment:
unneeded indefinite article in the grammar here, "a", should be `You now have access to this data`